src/data/queries/contactInfo.ts

```diff
@@ -0,0 +1,57 @@
+import { QueryParams, QueryFormatter } from 'next-drupal-query'
+import { ParagraphContactInformation } from '@/types/drupal/paragraph'
+import { ContactInfo } from '@/types/formatted/contactInfo'
+import { queries } from '.'
+
+export const params: QueryParams<null> = () => {
+  return queries.getParams().addInclude([
+    'field_additional_contact', // can be paragraph--phone_number or paragraph--email_contact
+    'field_contact_default',
+    'field_contact_default.field_office',
+    'field_benefit_hub_contacts',
+    'field_benefit_hub_contacts.field_support_services', // this is the additional contact info for a Benefit Hub, node--support_service
+  ])
+}
+
+// paragraph--contact_information is essentially a wrapper paragraph around several other entity references
+export const formatter: QueryFormatter<
+  ParagraphContactInformation,
+  ContactInfo
+> = (entity: ParagraphContactInformation) => {
+  const additionalContactType = entity.field_additional_contact?.type
+  let additionalContact = null
+
+  switch (additionalContactType) {
+    case 'paragraph--email_contact':
+      additionalContact = {
+        email: {
+          label: entity.field_additional_contact.field_email_label,
+          address: entity.field_additional_contact.field_email_address,
+        },
+      }
+      break
+    case 'paragraph--phone_number':
+      additionalContact = {
+        phone: {
+          label: entity.field_additional_contact.field_phone_label,
+          number: entity.field_additional_contact.field_phone_number,
+          extension: entity.field_additional_contact.field_phone_extension,
+        },
+      }
+      break
+  }
+
+  return {
+    contactType: entity.field_contact_info_switch,
+    defaultContact: {
+      title: entity.field_contact_default.title,
+      value: entity.field_contact_default.field_phone_number,
+      href: entity.field_contact_default.field_link.uri,
+    },
+    additionalContact,
```

Comment:
Some of the work I've done around the overarching structure of handling paragraphs includes the creation of a function `formatParagraph` that can take a paragraph of any type and return the formatted version. Since `field_additional_contact` is a field that can take variable types of paragraphs, I think this might eventually become:

`additionalContact: additionalContactType ? formatParagraph(entity.field_additional_contact) : null`

This, though, implies that `paragraph--phone_number` and `paragraph--email_contact` have their own formatter and templates. It seems like this PR actually moves away from that and removes email as its own standalone template. I'm curious about this.